plasmapy/physics/parameters.py

```diff
@@ -356,6 +358,10 @@ def thermal_speed(T, particle="e-", method="most_probable"):
         Method to be used for calculating the thermal speed. Options are
         `'most_probable'` (default), `'rms'`, and `'mean_magnitude'`.
 
+    mass : ~astropy.units.Quantity
+        The particle's mass. Defaults to None, but if set, overrides
```

Comment:
Looks like it's defaulting to `np.nan` instead of `None`.

Reply:
Yup, it is. My bad.